lib/src/conflicts.rs

```diff
@@ -239,6 +256,7 @@ pub fn materialize_merge_result(
                     output.write_all(
                         format!(" Conflict {conflict_index} of {num_conflicts}\n").as_bytes(),
                     )?;
+
```

Comment:
This newline is not a mistake  even if it's a bit out of place, I think it makes things a bit more readable. I can of course remove it if anybody prefers.